OpenSim/Tools/InverseKinematicsTool.cpp

```diff
@@ -144,57 +144,78 @@ void InverseKinematicsTool::setNull()
  */
 void InverseKinematicsTool::setupProperties()
 {
-    _modelFileNameProp.setComment("Name of the .osim file used to construct a model.");
+    _modelFileNameProp.setComment(
+        "Name of the model (.osim) to use for inverse kinematics.");
```

Comment:
I think you meant "Name of the model *file* (.osim)…" (to distinguish from the Model's `name` property).